pkg/model/workflow.go

```diff
@@ -195,7 +192,7 @@ func (j *Job) GetMatrixes() []map[string]interface{} {
 
 func commonKeysMatch(a map[string]interface{}, b map[string]interface{}) bool {
```

Comment:
Potentially, this entire function could be replaced with a `reflect.DeepEqual(a, b)`.
What do you think?